src/kaltura-player.js

```diff
@@ -772,18 +774,25 @@ class KalturaPlayer extends FakeEventTarget {
     this._localPlayer.detachMediaSource();
   }
 
-  _mergeProviderPluginsConfig(providerPluginsConfig: KPPluginsConfigObject): KPPluginsConfigObject {
+  mergeProviderPluginsConfig(providerPluginsConfig: KPPluginsConfigObject): KPPluginsConfigObject {
```

Comment:
don;t think we would want to make this a kaltura player API(removing the `_`)

Reply:
it's called from the playlist manager

Reply:
Still, not a valid API on the player, it is also externalized.

Reply:
if this becomes a helper function move it to a seperate utils and use it without side-effects.
As it is private you can change API to return the object with both providers config as before and also the cached app config and use the one you need where you need it.
Exposing this on the API is problematic as it will never be used and will just raise questions.